docs/components/concepts/clusters.md

```diff
@@ -6,72 +6,88 @@ description: "Learn more about the clusters available in your Camunda 8 plan."
 
-- Clusters tagged as `test`, `stage`, or `prod` do not auto-pause.
-- Paused clusters are automatically deleted after 30 consecutive paused days. You can change the tag to avoid cluster deletion.
-- No data is lost while a cluster is paused. All execution and configuration is saved, but cluster components such as Zeebe and Operate are temporarily disabled until you resume the cluster.
+:::info
+See [Camunda Enterprise General Terms](https://legal.camunda.com/licensing-and-other-legal-terms#camunda-enterprise-general-terms) for term definitions for **Monthly Uptime Percentage** and **Downtime**.
+:::
 
-:::tip
+## Cluster size
 
-To prevent auto-pause, you can:
+The cluster size defines the cluster performance and capacity.
 
-- Tag the cluster as `test`, `stage`, or `prod` instead of `dev`.
-- [Upgrade your Free Trial plan](https://camunda.com/pricing/) to a Starter, Professional, or Enterprise plan.
+- You can choose from three sizes of cluster (1x, 2x, 3x).
+- Each increase in size boosts cluster performance and adds capacity. Larger cluster sizes allow you to serve more workload.
+- Increased usage such as higher throughput or longer data retention requires a larger cluster size. See [sizing your environment](/components/best-practices/architecture/sizing-your-environment.md).
+- Each size increase uses one of your available cluster reservations.
 
-:::
+### Cluster size hardware packages
 
-## Development clusters
+Cluster sizes determine the hosting hardware package used for the cluster, as follows:
 
-Development clusters, available in the Starter and Enterprise plans, are recommended for development, testing, proof of concepts, and demos.
+| Size  | 1x  | 2x  | 3x  |
+| :---- | :-- | :-- | :-- |
+| Spec? |     |     |     |
 
-The way this type of cluster works varies depending on if you are using it in the Starter or the Enterprise plan.
+### Limitations
 
-### Development clusters in the Enterprise Plan
+- Currently, there are limits on the number of packages you can select.
+- Cluster sizes can only be increased for an existing cluster by contacting your Customer Success Manager.
 
-Enterprise Plan users can purchase development clusters as part of their Enterprise subscription agreement. Deployment and execution of models (process instances, decision instances, and task users) are included at no extra cost for this type of cluster. Additionally, this type of cluster in the Enterprise plan follows the [standard data retention policy](/components/concepts/data-retention.md) and does not auto-pause when not in use.
+## Free Trial clusters
```

Comment:
I think we need to keep Starter Plan docs separate 
@hisImminence I think for starter we dont do change right now? Can you confirm?
We can do this in interation 2 though.

Reply:
Keeping this info in here for now, but to remove/restructure at a later date.